tools/py/committee-reweight.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python3
+import sys
+import argparse
+import numpy as np
+from ipi.inputs.simulation import InputSimulation
+from ipi.utils.io.inputs import io_xml
+
+
+def direct_reweight(pot, obs, kbT):
+    """
+    reweights an observable based on a committee of potentials, at the temperature kT
```

Comment:
any reference to give here, how the reweighting is done in practice?

Reply:
Added a reference for the direct reweight scheme